File: DataServer/src/main/java/model/Avatar.java
package model;

public class Avatar
{
	private String Owner;
	private String AvatarName;
	private Status Status;
	private int TimeBlock;

	
	public Avatar(String owner, String avatarName, Status status, int timeBlock)
	{
		Owner = owner;
		AvatarName = avatarName;
		Status = status;
		TimeBlock = timeBlock;
	}
	
	public String getAvatarName()
	{
		return AvatarName;
	}
	
	public String getOwner()
	{
		return Owner;
	}
	
	public int getAcademic()
	{
		return Status.GetStatus("Academic");
	}
	
	public int getFinancial()
	{
		return Status.GetStatus("Financial");
	}
	
	public int getHealth()
	{
		return Status.GetStatus("Health");
	}
	
	public int getSocial()
	{
		return Status.GetStatus("Social");
	}

	public int getTimeBlock() {
		return TimeBlock;
	}
}

File: Client/StudentIncubator/Models/Avatar.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace StudentIncubator.Models
{
    public class Avatar
    {
        public string Owner { get; }
        public string AvatarName { get; set; }
        public Status Status { get; }
        public int TimeBlock { get; }
        public string EventDescription { get; }

        public Avatar(string owner, string avatarName)
        {
            Owner = owner;
            AvatarName = avatarName;
            Status = new Status(100, 100, 100, 100);
            TimeBlock = 80;
            EventDescription = null;
        }

        [JsonConstructor]
        public Avatar(string owner, string avatarName, Status status, int timeBlock, string eventDescription)
        {
            AvatarName = avatarName;
            Owner = owner;
            Status = status;
            TimeBlock = timeBlock;
            EventDescription = eventDescription;
        }
        
        public int GetAcademic()
        {
            return Status.GetStatus("Academic");
        }

        public int GetFinancial()
        {
            return Status.GetStatus("Financial");
        }

        public int GetHealth()
        {
            return Status.GetStatus("Health");
        }
        
        public int GetSocial()
        {
            return Status.GetStatus("Social");
        }
    }
}